examples/fmri_spm_auditory.py

```diff
@@ -315,7 +327,7 @@ def makelist(item):
 pipeline. Thus for this pipeline there will be subject specific
 sub-directories.
 
-The ``nipype.pipeline.engine.Pipeline.connect`` function creates the
+The :func:`nipype.pipeline.engine.Pipeline.connect` function creates the
```

Comment:
```suggestion
The :func:`nipype.pipeline.engine.Workflow.connect` function creates the
```

There's another one in the preceding paragraph.